cpcc2_tiago:

  urdf_path: {
    type: string,
    default_value: /home/jgleyze/stage_laas_gepetto/cpcc2_tiago_ws/src/cpcc2_tiago/urdf/tiago.urdf,
    description: "Path to the URDF file",
  }
  joints: {
    type: string_array,
    default_value: [ arm_1_joint, arm_2_joint, arm_3_joint, arm_4_joint, arm_5_joint], #it's very important for them to be ordered !
    description: "Name of the arm joints to control",
  }
  pveg_joints_command_type: {
    type: string_array,
    default_value: [ effort, effort, effort, effort, effort],
    description: "Type of command to send to the joints",
  }
  state_interfaces_name: {
    type: string_array,
    default_value : [position, velocity],
    description: "Name of the state interface",
  }

  OCP_horizon_length: {
    type: int,
    default_value: 20,
    description: "Horizon of the problem",
  }

  OCP_time_step: {
    type: double,
    default_value: 0.05,
    description: "OCP time step",
  }

  OCP_solver_iterations: {
    type: int,
    default_value: 1,
    description: "OCP iterations",
  }

  OCP_solver_frequency: {
    type: double,
    default_value: 20.0,
    description: "OCP solver frequency",
  }

  enable_file_logging: {
    type: bool,
    default_value: false,
    description: "Enable logging",
  }

  log_folder: {
    type: string,
    default_value: "CPCC2_log/",
    description: "Log folder",
  }

  enable_live_logging: {
    type: bool,
    default_value: true,
    description: "Enable data stream to topic for plotjuggler",
  }


